### ISSUE TITLE
dfCorTest and cyEffTest fail for mxn triangle

### ISSUE THREAD

**Issue by spencer-miller** [mages/ChainLadder#77]
The functions are set up to only handle nxn triangles. The AY correlation function appears to just need the value of n adjusted for years with one LDF. However, the CY test function may need more nuance.

**Comment by marcopark90** [comment]
Yes, you are right. As of now, the functions are set up only to handle symmetrical triangles. The implementations for non-symmetrical matrices are not yet developed.
